appveyor.yml

```diff
@@ -0,0 +1,51 @@
+environment:
+  PATH: C:\msys64\usr\bin;C:\msys64\mingw64\bin;C:\Windows\System32;C:\Windows;%PATH%
+  VSTEST: OFF
```

Comment:
I turned `VSTEST` to `OFF` because I'm seeing [test failures](https://ci.appveyor.com/project/donny-dont/portable/build/1.0.54/job/uhcfotgb51cmf4v1) with it on and fixing tests is outside of scope for this.

Results of `ctest` are
```bash
89% tests passed, 7 tests failed out of 64
Total Test time (real) =   8.77 sec
The following tests FAILED:
	 41 - pq_test (Failed)
	 48 - servertest (Failed)
	 53 - ssltest (Failed)
	 54 - testdsa (Failed)
	 55 - testenc (Failed)
	 56 - testrsa (Failed)
	 59 - tlstest (Failed)
Errors while running CTest
Command exited with code 8
```

Reply:
these appear to be all tests that have a batch file wrapper that drives an executable, so it might be a PATH or other issue with the test driver code.

Reply:
Adding '.' (current directory) appears to solve that.

```
PATH: .;C:\msys64\usr\bin;C:\msys64\mingw64\bin;C:\Windows\System32;C:\Windows;%PATH%
```

Reply:
That's why PATH 'Debug' is used in batch files.
In those days, I had no idea to pass build environment 'Release' or 'Debug' to test driver batch files.
So, with Debug build, regressions will be succeeded now, but with Release would be failed.